libcec/Makefile

```diff
@@ -43,4 +43,18 @@ define Package/libcec/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so* $(1)/usr/lib/
 endef
 
+
+define Package/cec-client
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=cec-client
+  DEPENDS:=+libcec
+endef
+
+define Package/cec-client/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) -L $(PKG_BUILD_DIR)/src/cec-client/cec-client $(1)/usr/bin/
+endef
+
 $(eval $(call BuildPackage,libcec))
+$(eval $(call BuildPackage,cec-client))
```

Comment:
Also maintain alphabetical order here.

Reply:
Will that work, given that cec-client is actually built by the libcec build step, and not separately? Im mean If `cec-client` is built before `libcec`, the binary to package would not yet exist? (Again, I have not enough insight into the complexity of the OpenWrt make setup to be sure).

Reply:
Hmmm you might be right. I'd expect the dependency to take care of that, but it might not be the case.